packages/m2m-gateway/src/services/eserviceService.ts

```diff
@@ -77,5 +77,44 @@ export function eserviceServiceBuilder(clients: PagoPAInteropBeClients) {
 
       return toM2MGatewayApiEServiceDescriptor(descriptor);
     },
+    async getEServiceDescriptors(
+      eserviceId: EServiceId,
+      {
+        state,
+        offset,
+        limit,
+      }: {
+        state?: m2mGatewayApi.EServiceDescriptorState;
+        offset: number;
+        limit: number;
+      },
```

Comment:
Why not typing it with `GetEServiceDescriptorsQueryParams`?
```suggestion
      }: m2mGatewayApi.GetEServiceDescriptorsQueryParams,
```

Reply:
It was because, for some reason, ESLint throws a warning if sum `offset + limit` without this explicit typing to `number` 🤷  But I'll try to disable the rule or something

Reply:
I managed to make TS / ESLint not complain by adding assertions on offset & limit